arrow/util.py

```diff
@@ -2,6 +2,9 @@
 from __future__ import absolute_import
 
 import datetime
+import numbers
+
+from arrow.constants import MAX_TIMESTAMP, MAX_TIMESTAMP_MS, MAX_TIMESTAMP_US
```

Comment:
I don't think we really need it since we have a lot of statically scoped variables, leaving only a set few constants. It is probably best to not clutter the global namespace and keep them statically scoped whenever possible.